Source/Shaders/PostProcessFilters/FXAA3_11.glsl

```diff
@@ -0,0 +1,2074 @@
+/**
```

Comment:
I stripped all of the unused shader code. There is a comment at the top that lists the steps. I added the original shader to `Source/ThirdParty`. It isn't included anywhere so it won't be part of the build, but I can remove it if necessary. There's also a link to the original in the trimmed version after the license.